src/App.tsx

```diff
@@ -25,16 +35,54 @@ function App() {
         <p>
           <input type='text' id="auth-code" placeholder='Paste authentication code here' />
           <button onClick={() => {
-            let code = (document.getElementById("auth-code") as HTMLInputElement).value;
-            invoke('finalize_auth', { code });
+            let code: string = (document.getElementById("auth-code") as HTMLInputElement).value;
```

Comment:
Would is be possible to use a `const` for `code` instead of `let`? When I see a variable defined as a let, I assume the intent of the author is to modify the value after initial assignment, but I think in this case, it doesn't change. Again, not a big deal, but it like a form of documentation how you declare your variables.